cypress/support/fragments/settings/circulation/titleLevelRequests.js

```diff
@@ -72,6 +75,28 @@ export default {
       });
   },
 
+  changeFailToCreateHoldForBlockedRequest(status) {
```

Comment:
method doesnt work if status 'allow'
method doesn't work if checkbox already selected

Reply:
used suggested approach
and added 
uncheckIfSelected() to the checkbox interactor since it needs to be unchecked